src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -139,7 +139,7 @@ public async Task RemoveAsync(IAccount account)
             if (AppConfig.IsBrokerEnabled && ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth())
             {
                 var broker = ServiceBundle.PlatformProxy.CreateBroker(null);
-                broker.RemoveAccountAsync(AppConfig.ClientId, account);
+                await broker.RemoveAccountAsync(AppConfig.ClientId, account).ConfigureAwait(false);
```

Comment:
Hmm, do we have a logger without the requestContext? We should add a bit of logging here...

Reply:
Ah, you might be right. let me double check

Reply:
The logger will be injected by the platform proxy so everything will be logged